lib/ex_huobi/futures/private.ex

```diff
@@ -0,0 +1,26 @@
+defmodule ExHuobi.Futures.Private do
```

Comment:
Unit tests for this also

Reply:
Can you add the typespec for all of the public functions https://hexdocs.pm/elixir/typespecs.html